docs/source/reference-material/gallery.rst

```diff
@@ -1,27 +1,36 @@
-Examples
-========
-
+Gallery
```

Comment:
I'd rename this to "Code Examples".

Gallery makes it hard to know what this tab is supposed to be without clicking on it.